lib/model/autocomplete.dart

```diff
@@ -337,13 +429,25 @@ class MentionAutocompleteQuery {
 }
 
 class AutocompleteDataCache {
+  final Map<int, String> _namesByUser = {};
+
+  /// The lowercase `fullName` of [user].
+  String nameForUser(User user) {
```

Comment:
```suggestion
  /// The lowercase `fullName` of [user].
  String normalizedNameForUser(User user) {
```

It's not quite the name, but rather a version of the name that's been processed into a normalized form useful for autocomplete. So we can have the method's name distance itself a bit from saying it's simply the user's name.